Dockerfile

```diff
@@ -4,6 +4,22 @@ ARG FLEET_VERSION=papeeria-compiler-fleet-1.0-SNAPSHOT
 
 ADD ./build/distributions/${FLEET_VERSION}.tar  /
 
+RUN apt-get -qq update && apt-get -qqy install wget
+RUN wget -qO- https://get.docker.com/ | sh
+
+# Macos has weird permissions, tmp allows to read & write without root user
```

Comment:
We don't care about MacOS. This is supposed to be run on Linux, and there is no problem with mounting any directory.